packages/glimmer-test-helpers/lib/environment.ts

```diff
@@ -681,6 +708,10 @@ export class TestEnvironment extends Environment {
     return modifier;
   }
 
+  compileSpec(template: string) {
+    return compileRealSpec(template, { env: this });
```

Comment:
This can be inlined into `registerPartial` (it doesn't need to be its own method)